### PR TITLE
Feature/refactor sankey layout

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR:

- Moves the tool page templates to a new react component
- Changes title bar vanilla component into a react component
- Fixes the feedback not showing in most of the pages and uses a shared react component

I didn't go too deep with the changes in the layout because of the SET_SANKEY_SIZE action which has many dependencies in the whole legacy Sankey layout

### CODE REVIEW

File: frontend/scripts/pages/tool.page.jsx [hunk 1]
```diff
@@ -1,95 +1,7 @@
...
-    document.getElementById('js-react-vanilla-bridge-container')
-  );
-
-  evManager.addEventListener(window, 'resize', () => resizeSankeyTool(store.dispatch));
-  document.querySelector('body').classList.add('-overflow-hidden');
-};
-
-export const unmount = () => {
-  evManager.clearEventListeners();
-  unmountComponentAtNode(document.getElementById('js-tool-nav-react'));
```
Comment (sorodrigo) [inline frontend/scripts/pages/tool.page.jsx]: we need to unmount the `<ToolLayout />`

File: frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+      <MapContextContainer />
+      <NodesTitlesContainer />
+      <Sankey />
+      <TooltipContainer />
+      <ModalContainer />
+    </>
+  );
+
+  return (
+    <Provider store={store}>
```
Comment (sorodrigo) [inline frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx]: I prefer moving the provider out of the component

File: frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+      <div>
+        <div className="js-node-tooltip c-info-tooltip" />
+        <div className="js-sankey-tooltip c-info-tooltip" />
+        <div className="loading-wrapper">
```
Comment (sorodrigo) [inline frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx]: we don't need the loading wrapper anymore

File: frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+          </g>
+        </g>
+      </svg>
+    </div>
+
+    {renderExpandButton()}
+  </div>
+);
+
+const ToolLayout = ({ store }) => {
```
Comment (sorodrigo) [inline frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx]: Could we call it `Tool`? in the rest of the pages the layout is called `${parentDirectory}.component.js`

Comment (Bluesmile82) [reply frontend/scripts/react-components/tool/tool-layout/tool-layout.component.jsx]: Actually, I'm going to move it directly to the react-components/tool directory. This is the main component and it should be at the root

**Comment by Bluesmile82** [comment]
Ready again! Thanks for the comments

**Pull request merged**